src/rpcblockchain.cpp

```diff
@@ -1620,7 +1615,7 @@ Value execute(const Array& params, bool fHelp)
         std::string timestamp = TimestampToHRDate(mvApplicationCacheTimestamp["superblock;magnitudes"]);
         entry.push_back(Pair("Superblock Timestamp",timestamp));
         entry.push_back(Pair("Superblock Block Number",mvApplicationCache["superblock;block_number"]));
-        double height = cdbl(ReadCache("neuralsecurity","pending"),0);
+        int height = std::stoi(ReadCache("neuralsecurity","pending"));
```

Comment:
Is this safe? Remember that when string is converted to number things can always go bad.

Reply:
worked correctly when tested and ravon mentioned this. and cdbl just casts this to double and i dont see a point in have 8 decimal places of 0s it seems rather messy.

Reply:
if u got a better suggestion that doesn't have decimal places im all ears. the readcache and writecache uses string

Reply:
ah i guess i can leave that as a string. why convert it.  just a rpc response there